pkg/relayer/proxy/proxy_test.go

```diff
@@ -112,19 +137,19 @@ func TestRelayerProxy_NoProxiedServices(t *testing.T) {
 }
 
 // RelayerProxy should fail to start if it cannot spawn a server for the
-// services it advertized on-chain
-// TODO_TECHDEBT: Re-enable this test once the RelayerProxy is building its
-// servers from the provided config file
+// services it advertized on-chain.
+// TODO_TECHDEBT(@okdas, @red-0ne): Re-enable this test once the RelayerProxy
+// is building its servers from the provided config file.
 //func TestRelayerProxy_UnsupportedRpcType(t *testing.T) {
```

Comment:
Can you do the following.

1. Uncomment the test
2. Use `t.skip("")` in the very beginning of the test so it's not executed
3. Add the TODO comment inside the message inside t.skip

Reply:
The test no longer depends on that TECHDEBT, will renable it for now.